corehq/apps/reports/dbaccessors.py

```diff
@@ -12,4 +13,5 @@ def get_exports(domain):
         startkey=startkey,
         endkey=endkey,
         include_docs=True,
+        stale=settings.COUCH_STALE_QUERY,
     )
```

Comment:
I think I'd add a `strict` arg to make it an explicit part of this function's interface that it's allowed to return slightly out of date data if `strict=False`